recipe/meta.yaml

```
@@ -20,10 +19,9 @@ requirements:
     - setuptools
     - pip
     - wheel
-
   run:
     - python
-    - aiobotocore 2.4.*
+    - aiobotocore 2.4.2
```

Based on tilde use [here](https://github.com/fsspec/s3fs/blob/2023.3.0/requirements.txt#L1), it should be:

```suggestion
    - aiobotocore >=2.4.2,<2.5
```

---

The reason I used exact version is because there is nothing between 2.4.2 and 2.5, which is already released. Should I still use the suggested range, @cbouss ?

---

The range should be used because this is what is defined by upstream.

Moreover, what matters is not only what is currently released, but also the promise of what will be released in the future (i.e. the promise that aiobotocore will not introduce breaking changes between minor versions).

---

Thanks for clarification.